pkg/resources/external_volume_acceptance_test.go

```diff
@@ -0,0 +1,1854 @@
+package resources_test
+
+// TODO Add test that includes Iceberg table creation, as this impacts the describe output (updates ACTIVE)
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/assert"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/assert/resourceassert"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
+	r "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+var (
+	comment          = random.Comment()
+	comment2         = random.Comment()
+	allowWritesTrue  = "true"
+	allowWritesFalse = "false"
+)
+
+var (
+	s3StorageLocationName  = "s3Test"
+	s3StorageLocationName2 = "s3Test2"
+	s3StorageProvider      = "S3"
```

Comment:
`S3GOV` acceptance tests are missing as I don’t have an awsgov account (same situation as in sdk integration tests).